taxonomies/categories.txt

```diff
@@ -21922,6 +22022,12 @@ nl:Amandelchocoladen
 ru:Шоколад с миндалём
 se:Choklad med mandlar, choklad med mandel
 
+<en:Chocolates
+en:Toblerone milk cholcolate bar w nougat
```

Comment:
We shouldn't put brands or abbreviations (like "w").
I suggest "Milk chocolate bar with nougat"  --> notice I fixed the Ciqual error in chocolate :)

Reply:
and I forgot : all our categories, in French and in English, should be in plural

Reply:
I added Toblerone as a synonym. Remove completely ?